src/main/public/js/expandable-checkbox-option.js

```diff
@@ -0,0 +1,96 @@
+$(document).ready(function () {
```

Comment:
in future I don't think we should write new code in jquery, govuk-frontend doesn't require it and all the browsers that you might need it for are out of support